README.md

```diff
@@ -54,6 +54,18 @@ If you had a `jar:test-jar` execution, delete it and add to `properties`:
 <no-test-jar>false</no-test-jar>
 ```
 
+To fix spotless validation errors:
```

Comment:
may want to mention this: https://github.com/jenkinsci/github-branch-source-plugin/blob/master/.git-blame-ignore-revs

(may be more appropriate in release notes)